code/modules/antagonists/traitor/contractor/datums/syndicate_contract.dm

```diff
@@ -496,44 +490,42 @@
   * * M - The target mob.
   */
 /datum/syndicate_contract/proc/injure_target(mob/living/M)
-	if(!prob(RETURN_INJURY_CHANCE) || M.health < 50)
-		return
-
 	var/obj/item/organ/external/injury_target
-	if(prob(20)) //remove a limb
-		if(prob(50))
-			injury_target = M.get_organ(pick(BODY_ZONE_PRECISE_R_HAND, BODY_ZONE_PRECISE_L_HAND, BODY_ZONE_PRECISE_R_FOOT, BODY_ZONE_PRECISE_L_FOOT))
-			if(!injury_target)
-				default_damage(M)
-				return
-			injury_target.droplimb()
-			to_chat(M, "<span class='warning'>You were interrogated by your captors before being sent back! Oh god something's missing!</span>")
-	else //fracture
-		if(ismachineperson(M))
-			M.emp_act(EMP_HEAVY)
-			M.adjustBrainLoss(30)
-			to_chat(M, "<span class='warning'>You were interrogated by your captors before being sent back! You feel like some of your components are loose!</span>")
-
-		else if(isslimeperson(M))
-			injury_target = M.get_organ(pick(BODY_ZONE_PRECISE_R_HAND, BODY_ZONE_PRECISE_L_HAND, BODY_ZONE_PRECISE_R_FOOT, BODY_ZONE_PRECISE_L_FOOT))
-			if(!injury_target)
-				default_damage(M)
-				return
-			injury_target.cause_internal_bleeding()
-
-			injury_target = M.get_organ(BODY_ZONE_CHEST)
-			injury_target.cause_internal_bleeding()
-			to_chat(M, "<span class='warning'>You were interrogated by your captors before being sent back! You feel like your inner membrane has been punctured!</span>")
-
-		if(prob(25))
-			injury_target = M.get_organ(BODY_ZONE_CHEST)
-			injury_target.fracture()
-		else
-			injury_target = M.get_organ(pick(BODY_ZONE_PRECISE_R_HAND, BODY_ZONE_PRECISE_L_HAND, BODY_ZONE_R_LEG, BODY_ZONE_R_LEG))
-			if(!injury_target)
-				default_damage(M)
-				return
-			injury_target.fracture()
+	if(prob(20)) //See if they're !!!lucky!!! enough to just chop a hand or foot off first
+		injury_target = M.get_organ(pick(BODY_ZONE_PRECISE_R_HAND, BODY_ZONE_PRECISE_L_HAND, BODY_ZONE_PRECISE_R_FOOT, BODY_ZONE_PRECISE_L_FOOT))
+		if(!injury_target)
+			return
+		default_damage(M)
+		injury_target.droplimb()
```

Comment:
The `!injury_target` guard clause here means that if you happen to be missing the randomly selected limb, you just avoid the default damage - is this intentional? Same thing happens for Slimepeople and the universal limb fracture+IB later

Reply:
This was the behavior it had before which I kept in since I thought it could make amputees get lucky sometimes, but you're right that the basic damage should probably be _guaranteed_ guaranteed, considering that's the point of the PR.

Reply:
Oh, wait it was actually to prevent runtimes when trying to damage a limb that didn't exist.